pandas/tests/base/test_misc.py

```diff
@@ -151,6 +151,15 @@ def test_searchsorted(request, index_or_series_obj):
         #  comparison semantics https://github.com/numpy/numpy/issues/15981
         mark = pytest.mark.xfail(reason="complex objects are not comparable")
         request.node.add_marker(mark)
+    elif any(isinstance(elem, int) for elem in obj.values[:]) and any(
+        isinstance(elem, str) for elem in obj.values[:]
```

Comment:
this is a pretty awkward check.  maybe just check obj[0] and obj[1] with a comment the check is written with the mixed-int-string entry in mind?

Reply:
agreed, I replaced the check with `tm.assert_mixed_int_string_entry`